src/components/dialogs/field-hook.js

```diff
@@ -80,6 +80,11 @@ export const useTextValue = ({
     return [value, field, setValue, hasChanged];
 };
 
+const makeAdornmentEndIcon = (content) => {
+    return {
+        endAdornment: <InputAdornment position="end">{content}</InputAdornment>,
```

Comment:
Why you put the JSX into a JSON ?

Reply:
I reset the current code on main branch. thank you